README.md

```diff
@@ -195,11 +195,10 @@ An [Effect] are the instruction to the [Executor] to run set of [Task]s concurre
 
 ```ipldsch
 type Task struct {
-  on      URI
-  call    Ability
-  input   {String : Any}
-
-  nnc     string
+  obj    URI
+  call   Ability
+  args   {String : Any}
```

Comment:
personally, I'd go with `input` still or `params` as a more expressive word in function land. `args` align to closely w/ scripts for me, where `params` expressed function params better?

Reply:
I prefer `input` as well otherwise `params`. Args in my head implies positional parameters as opposed to named.

Reply:
Changing back to `input` 👍